src/Spreadsheet/DeliverySpreadsheetParser.php

```diff
@@ -174,6 +178,17 @@ private function enhanceTask(Task $task, string $prefix, array $record)
         }
     }
 
+    private function applyTags(TaggableInterface $task, $tagsAsString)
+    {
+        $tagsAsString = trim($tagsAsString);
```

Comment:
We may want to better sanitize the user input

```suggestion
        $tagsAsString = preg_replace("/[[:blank:]]+/", " ", trim($tagsAsString));
```